.circleci/smoke.sh

```diff
@@ -23,6 +23,7 @@ echo "Setting up Fury in $TMPDIR"
 create_package
 install fury
 install fury-adapter-apib-parser
+install fury-adapter-swagger
```

Comment:
fury-adapter-swagger was not included in smoke tests and thus failed CI: https://circleci.com/gh/apiaryio/api-elements.js/3712?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

That's because it tried to install from NPM (and its unreleased).